banzai_nres/utils/fits_utils.py

```diff
@@ -0,0 +1,25 @@
+from astropy.io import fits
+
+
+def fits_bytes_to_header(header_lines):
```

Comment:
minor comment, but if this only is meant for pheonix headers then this should be renamed to something like pheonix_bytes_to_header, on first read through I thought this was a general function and was confused.